### DISCUSSION TITLE
how to use powershell

### DISCUSSION THREAD

**Discussion by bryant-the-coder** [wezterm/wezterm#2091] [General]

**Comment by wez** [comment]
Take a look at https://wezfurlong.org/wezterm/config/launch.html for information on how to launch programs

**Comment by bryant-the-coder** [comment]
oh btw i cant seem to set operatormonolig nerd font :/

**Reply by MuhammedZakir** [reply]
Please post:
1. what you tried, including your config.
3. what you expected.
3. what actually happened.

The way you post, it's impossible to determine what's the problem. Don't take this as personal; it's just that without proper info, we really can't help you, and even if we tried, the answers you get will probably be unrelated or wrong. :-)

**Reply by bryant-the-coder** [reply]
![image](https://user-images.githubusercontent.com/92417638/172507492-02ea98b3-5dc3-4131-be52-e6cddc5d199c.png)

**Reply by bryant-the-coder** [reply]
@MuhammedZakir

**Reply by MuhammedZakir** [reply]
I looked into it, but was stumped even before testing as Operator Mono font is proprietary. You'll have to wait for @_wez_ to know about possible reasons for why you're getting that error.

Some suggestions:
1. `wezterm ls-fonts` command may help you. Try: `wezterm ls-fonts --list-system | sed -n '/wezterm.font.*Operator/p'`.

2. Because WezTerm has font-fallback support, I recommend you to take advantage of it for Nerd icons and ligatures. It's much better than using a patched font. Also, fyi, WezTerm already ships with `JetBrains Mono` which has ligatures and `Symbols Nerd Font Mono` which is Nerd-icons-only font.

Edit: Change `grep` to `sed` for filtering. `grep` gives partial result.

When you list the fonts using `wezterm ls-fonts --list-system`, is `OperatorMonoLig Nerd Font` with same attributes shown more than once? I was able to get the same error as yours using `Terminus` font. When listing fonts using `wezterm`, `Terminus` font is shown like this:
```
wezterm.font("Terminus", {weight="Regular", stretch="Normal", style="Normal"}) -- /usr/share/fonts/misc/ter-x12b.pcf.gz, FontConfig pixel_sizes=[12]
wezterm.font("Terminus", {weight="Regular", stretch="Normal", style="Normal"}) -- /usr/share/fonts/misc/ter-x12n.pcf.gz, FontConfig pixel_sizes=[12]

--- upto pixel_sizes=[32] ---
```

I suspect this to be a bug.

**Reply by wez** [reply]
Nerd Font patched fonts often break the name information of the font and cause problems with font resolving. My money would be on the name not being what you expect. `wezterm ls-fonts --list-system | grep Operator` should show you nearby examples of font names that should work.

You shouldn't need to use a patched font as wezterm has a default fallback to an embedded version of the Nerd Font symbols font.  I'd recommend just using Operator Mono Lig (that's what I use).

**Reply by bryant-the-coder** [reply]
noted with thanks :)

**Reply by bryant-the-coder** [reply]
btw how to make wezterm start on fullscreen?

**Reply by bryant-the-coder** [reply]
@MuhammedZakir

**Reply by MuhammedZakir** [reply]
Not implemented, see https://github.com/wez/wezterm/issues/284.
